pandas/tests/extension/base/interface.py

```diff
@@ -86,3 +87,25 @@ def test_isna_extension_array(self, data_missing):
             assert not na.all()
 
             assert na.dtype._is_boolean
+
+    @pytest.mark.parametrize('periods, indices', [
```

Comment:
I would move this to methods.py I think (and obviously move the overrides in arrow as well)

Reply:
Since moving this to methods takes it out of the `BaseInterfaceTests` class, I can discard the overrides in arrow, since, currently, there is no test class within it deriving from `BaseMethodsTests` (or `BaseExtensionTests` for that matter). Is that acceptable?

Reply:
see my comment below